tools/deployment/package-helm/templates/otel-collector-deployment.yaml

```
@@ -0,0 +1,35 @@
+  replicas: 1
+  selector:
+    matchLabels:
+      app: otel-collector
+  template:
+    metadata:
+      labels:
+        app: otel-collector
+    spec:
+      containers:
+        - name: otel-collector
+          image: {{ .Values.telemetry.collector.image | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)